kuma/settings/common.py

```diff
@@ -1452,3 +1470,23 @@ def get_user_url(user):
 
 # Tell django-taggit to use case-insensitive search for existing tags
 TAGGIT_CASE_INSENSITIVE = True
+
+# Content Experiments
+# Must be kept up to date with PIPELINE_JS setting and the JS client-side
+#  configuration. The 'id' should be a key in PIPELINE_JS, that loads
+#  Traffic Cop and a client-side configuration like
+#  kuma/static/js/experiment-wiki-content.js
+#
```

Comment:
For safety's sake, may be worth adding a comment here that only one experiment should be active for a given `locale` + `slug`.